### ISSUE TITLE
Possible to keep some of the old apt packages when an upgrade rolls out?

### ISSUE THREAD

**Issue by VacuousCoder** [timescale/timescaledb#1366]
**Relevant system information:**
 - OS: Ubuntu/Debian - 16.04 tested
 - PostgreSQL version (output of `postgres --version`): 11 tested - others probably
 - TimescaleDB version (output of `\dx` in `psql`): all
 - Installation method: apt

**Describe the bug**
I'm seeing a trend of breaking changes from version to version of timescale that prevents an import from a lower version to a higher version. There is currently no way for me to take a database dump from production on say 1.3.2, spin up a vm, install timescale, load the backup, and upgrade to 1.4.0 using apt. Currently an `apt-cache madison timescaledb-postgresql-11` shows only 1.4.0. It would be nice if you could roll the last few releases so that people can ensure upgrades will go smoothly before having to attempt them on prod. 

**To Reproduce**
1. Install timescale using apt on 16.04
2. Wait for a new version to release with breaking changes
3. Go to test the upgrade using a backup
4. Realize you can't install an old version from apt

**Expected behavior**
Allow people to install old packages for a while 
```
root@host:~# apt-cache madison postgresql
postgresql | 11+203.pgdg16.04+1 | http://apt.postgresql.org/pub/repos/apt xenial-pgdg/main amd64 Packages
postgresql | 11+203.pgdg16.04+1 | http://apt.postgresql.org/pub/repos/apt xenial-pgdg/main i386 Packages
postgresql | 9.5+173ubuntu0.2 | http://us.archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages
postgresql | 9.5+173ubuntu0.2 | http://us.archive.ubuntu.com/ubuntu xenial-updates/main i386 Packages
postgresql | 9.5+173ubuntu0.1 | http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages
postgresql | 9.5+173ubuntu0.1 | http://security.ubuntu.com/ubuntu xenial-security/main i386 Packages
postgresql |    9.5+173 | http://us.archive.ubuntu.com/ubuntu xenial/main amd64 Packages
postgresql |    9.5+173 | http://us.archive.ubuntu.com/ubuntu xenial/main i386 Packages
```

**Comment by VacuousCoder** [comment]
Closing because I dug into your repo and found what I needed.

To anyone else looking you can install old versions like this
`apt-get install timescaledb-1.3.2-postgresql-11`

Browse the repo here

https://launchpad.net/~timescale/+archive/ubuntu/timescaledb-ppa

**Comment by RobAtticus** [comment]
Hi @VacuousCoder -- I wanted to let you know that also when you install `timescaledb-postgresql-11` it actually contains the `.so` files for the past 10 releases. In this case, 1.4.0 contained `.so` files for every release starting with `1.0.0`. 

However I guess that does not help you if you are running this test on a new box, in which case, yes, you need to download the versioned package as you found.

**Comment by VacuousCoder** [comment]
Oh ok. I missed that. I installed the package specified and ran this

`CREATE EXTENSION timescaledb WITH VERSION '1.3.2';`

I did notice that installing `timescaledb-1.3.2-postgresql-11` uses `timescaledb-loader-postgresql-11` which doesn't have a release tied to it which currently pulls `1.4.0~ubuntu16.04` to be used by the old 1.3.2. I believe this is why it will attempt to load 1.4.0 even though 1.3.2 is the only one available when installing the specific 1.3.2 package. 

So, this fails
```
test=# CREATE EXTENSION timescaledb;
ERROR:  extension "timescaledb" has no installation script nor update path for version "1.4.0"
```

However, this works fine. 

```
test=# CREATE EXTENSION timescaledb WITH VERSION '1.3.2';
WARNING:
WELCOME TO
```
